marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults

```diff
@@ -0,0 +1,7 @@
+## Edit this file to customize your configuration ##
+fs-exporter.target=.
+
+# Set list of templates to include, comma separated values.
+nuxeo.template.includes=common
+
+nuxeo.fs-exporter.plugin.class=org.nuxeo.io.fsexporter.DefaultExporterPlugin
```

Comment:
why do you put the class in the template?

(btw missing blank line at end of file)

Reply:
For the default value of the contribution. That way, the OPS just have to update the `nuxeo.conf` file to update the plugin behavior.

Reply:
It's better to have the default in the code than in the config. Putting classes in config is something we should avoid.

Reply:
The thing is that the configuration is in an extension point. It has to be configurable by the ops team thru `nuxeo.conf`. We could put an indirection name -> class but for the current need, it's really not worth it. We will probably have to rewrite that whole module at some point.

Reply:
Moved as a comment and changed template in `nxftl` to better handle default case.